returnn/torch/engine.py

```diff
@@ -138,6 +158,11 @@ def init_train_from_config(
         self._train_step_func = self.config.typed_value("train_step")
         assert self._train_step_func, "train_step not defined"
 
+        self._gradient_accumulation_steps = config.int("gradient_accumulation_steps", 1)
```

Comment:
In the TF engine, the option is called `accum_grad_multiple_step`. Why do you call it different here? I think we should just use the same name.